app/models/ems_cluster.rb

```diff
@@ -190,7 +190,7 @@ def remove_all_resource_pools
 
   # All RPs under this Cluster and all child RPs
   def all_resource_pools
-    descendants(:of_type => 'ResourcePool')[1..-1].sort_by { |r| r.name.downcase }
+    descendants(:of_type => 'ResourcePool').select { |r| !r.is_default }.sort_by { |r| r.name.downcase }
```

Comment:
@agrare is this correct?  does the default resource pool not count here?

Note, this might the the bug in ancestry [here](https://github.com/stefankroes/ancestry/issues/217) @Fryguy opened based on my bad attempt to debug it [here](https://github.com/ManageIQ/manageiq/pull/1210)